lib/build.js

```diff
@@ -455,19 +455,21 @@ const parseAct = config => (act_str, silent = false) => {
                  * preceded by:
                  *  - ', ' or ','
                  *  - '('
+                 *  - '['
                  *  - the start of a line
                  *  - a tab
                  *  - a space
                  * and followed by:
                  *  - ','
                  *  - ')'
+                 *  - ']'
                  *  - the end of a line
                  *  - a tab
                  *  - a space
                  *
-                 *  example here: https://regex101.com/r/ZubkN1/3
+                 *  example here: https://regex101.com/r/ZubkN1/4
                  */
-                "from": new RegExp(`(, ?|\\(|^|\\t| )(${l.split(":=")[0].trim()})(,|\\)|$|\\t| )`, "gm"),
+                "from": new RegExp(`(, ?|\\(|\\[|^|\\t| )(${l.split(":=")[0].trim()})(,|\\)|$|\\t|\\]| )`, "gm"),
```

Comment:
Could you explain this regex to me?

This is matching the LHS of a `where` assignment, right? Like

```act
where
  A := B
```

When does the LHS contain brackets etc? I feel like I'm missing something.

Reply:
the brackets are capture groups. It's the only way I could find to make the regex work nice. I would be happy if you know how to do it without.

Reply:
There is a fairly complete set of examples for the regex [here](Could you explain this regex to me?). This is documented in the comment above.